### DISCUSSION TITLE
[designspacelib] 'STAT' Axis Value Table format 4 ?

### DISCUSSION THREAD

**Discussion by BlackFoundry** [fonttools/fonttools#2820] [Q&A] [answered]
Is there a known way of adding an axis value table format 4 to a VF with designspacelib, and in particular how is it implemented in designspace v5 specifications?

https://learn.microsoft.com/en-us/typography/opentype/spec/stat#axis-value-table-format-4

From this `getFormat` function it looks like there is no support for it in designspacelib yet:
https://github.com/fonttools/fonttools/blob/796c1f13303f0af515166e95bff1191f8d4e7f16/Lib/fontTools/designspaceLib/__init__.py#L1095

How would we define its description in a .designspace document knowing that the `<label>` has to refer to several axes locations?
Currently in designspace v5 the `<labels>` belongs to the `<axis>` enabling only desciption of formats 1, 2 and 3. But for axis value table format 4, the label has to refer to several axes locations combined…

**Comment by justvanrossum** [comment, marked as answer]
This should be it: https://fonttools.readthedocs.io/en/latest/designspaceLib/xml.html#labels-element-top-level

Example: https://github.com/fonttools/fonttools/blob/main/Tests/designspaceLib/data/test_v5.designspace#L56-L74

**Reply by BlackFoundry** [reply]
That's great! just what I needed! Thanks!
